src/index.ts

```diff
@@ -22,33 +20,6 @@ const app = express()
 app.use(cors())
 const router = express.Router()
 
-/** Bedtime Routes */
-
-router.get(
-  ['/embed/api/tracks/:id', '/embed/api/tracks/hashid/:hashId'],
-  (req: express.Request, res: express.Response) => {
-    getBedtimeResponse(BedtimeFormat.TRACK, req, res)
-  }
-)
-
-router.get(
-  ['/embed/api/collections/:id', '/embed/api/collections/hashid/:hashId'],
-  (req: express.Request, res: express.Response) => {
-    getBedtimeResponse(BedtimeFormat.COLLECTION, req, res)
-  }
-)
```

Comment:
do we want to remove the track and collection endpoints also? or just collectibles?

Reply:
they're unused @nicoback2 @nicoback  moved these to `@audius/sdk` a bit ago

Reply:
ah nice